web3_proxy/src/frontend/authorization.rs

```diff
@@ -1308,11 +1262,11 @@ impl Web3ProxyApp {
                         // TODO: Do the logic here, as to how to treat the user, based on balance and initial check
                         // Clear the cache (not the login!) in the stats if a tier-change happens (clear, but don't modify roles)
                         if let Some(downgrade_user_tier) = user_tier_model.downgrade_tier_id {
-                            let balance = latest_balance.read().clone();
+                            let balance = latest_balance.read().await.clone();
 
                             // only consider the user premium if they have paid at least $10 and have a balance > $.01
                             // otherwise, set user_tier_model to the downograded tier
-                            if balance.total_deposit < Decimal::from(10)
+                            if balance.total_deposits > Decimal::from(10)
```

Comment:
Why flip this? I'm pretty sure it was correct. We want to use the downgrade tier id in two cases. 

1. They haven't yet paid $10
2. They have paid >$10 but their balance is < $0.01

Reply:
ah ok that#s what it is, sorry for some reason I read the || as an &&, thus the confusion. Thanks! I changed it back